code/modules/mob/living/carbon/carbon.dm

```diff
@@ -19,6 +19,10 @@
 /mob/living/carbon/Destroy()
 	QDEL_LIST(internal_organs)
 	QDEL_LIST(stomach_contents)
+	var/mob/living/simple_animal/borer/B = has_brain_worms()
```

Comment:
we don't want to do:

```
if(borer)
	borer.leav_host()
	QDEL_NULL(borer)
```

Because then `borer` will be `null` and won't be `qdel`'d.